install_files/securedrop-app-code/debian/control

```diff
@@ -6,5 +6,5 @@ Homepage: https://securedrop.org
 
 Package: securedrop-app-code
 Architecture: amd64
-Depends: python-pip,apparmor-utils,gnupg2,haveged,python,secure-delete,sqlite3,${dist:Depends},libapache2-mod-wsgi,libapache2-mod-xsendfile,redis-server,supervisor,securedrop-keyring,securedrop-config,libpython2.7-dev
+Depends: python3-pip,apparmor-utils,gnupg2,haveged,python3,secure-delete,sqlite3,${dist:Depends},libapache2-mod-xsendfile,redis-server,supervisor,securedrop-keyring,securedrop-config,devscripts
```

Comment:
In order to avoid upstream updates to `libapache2-mod-wsgi` breaking our customizations to the `/etc/apache2/mods-available/wsgi.load`, we should pin the current-latest version of `libapache2-mod-wsgi` in the dependencies. This is a temporary measure—we still intend to remove that package later.

Reply:
Pinned. I think I got the version specification correct, as securedrop-app-code is happily installing, but please double check.